frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -81,69 +89,106 @@ async function deleteUserRoleAssignment(
-                @deleteUserRoleAssignment="deleteUserRoleAssignment"
-            />
-        </div>
-    </div>
+		<div class="dashboard-background-layout">
+			<NotificationStack />
+			<TabView
+				:pt="{
+					root: {
+						style: `${isApplicationSelected ? 'margin-top: 1.5rem' : ''}`
+					},
+					nav: {
+						style: `${isApplicationSelected ? 'margin: 0.112rem 2.55rem 0;' : 'display: none'}`,
+					},
+					panelContainer: {
+						style: `${isApplicationSelected ? 'margin-top: 2.62rem;' : ''}`,
+					},
+				}"
+			>
+				<TabPanel
+					header="Users"
+
+				>
+					<template #header>
+						<Icon
+							icon="user"
+							:size="IconSize.small"
+						/>
+					</template>
+					<UserDataTable
+						:isApplicationSelected="isApplicationSelected"
+						:loading="LoadingState.isLoading.value"
+						:userRoleAssignments="userRoleAssignments || []"
+						:selectedApplicationDisplayText="selectedApplicationDisplayText"
+						@deleteUserRoleAssignment="deleteUserRoleAssignment"
+					/>
+				</TabPanel>
+				<TabPanel
+					header="Delegated admins"
+					:disabled="true"
+
+				>
+					<template #header>
+						<Icon
+							icon="enterprise"
+							:size="IconSize.small"
+						/>
+					</template>
+				</TabPanel>
+			</TabView>
+		</div>
+	</div>
 </template>
 
 <style scoped lang="scss">
 @import '@/assets/styles/base.scss';
 
 .application-group {
-    display: grid;
+	display: grid;
 
-    label {
-        margin-bottom: 0.5rem;
-    }
+	label {
+		margin-bottom: 0.5rem;
+	}
 }
 
 .application-dropdown {
-    max-width: calc(100vw - 3rem);
-    height: 3rem;
-    padding: 0;
+	max-width: calc(100vw - 3rem);
+	height: 3rem;
+	padding: 0;
 
-    &:deep(.p-dropdown-label) {
-        padding: 0.8375rem 1rem;
-    }
+	&:deep(.p-dropdown-label) {
+		padding: 0.8375rem 1rem;
+	}
 }
 
 .dashboard-background-layout {
-    margin-top: 3rem;
-    padding: 1rem 0rem;
-    background: $light-layer-one;
-    z-index: -1;
-    margin-left: -2.5rem !important;
-    min-height: calc(100vh - 19.125rem) !important;
-    width: calc(100vw + 3rem) !important;
+	margin-top: 3rem;
+	padding: 1rem 0rem;
+	background: $light-layer-one;
+	z-index: -1;
+	margin-left: -2.5rem !important;
+	min-height: calc(100vh - 19.125rem) !important;
+	width: calc(100vw + 3rem) !important;
 }
 
 @media (min-width: 495px) {
-    .application-dropdown {
-        max-width: 29rem;
-    }
+	.application-dropdown {
+		max-width: 29rem;
+	}
 }
 
 @media (min-width: 768px) {
-    .dashboard-background-layout {
-        width: 100vw !important;
-    }
+	.dashboard-background-layout {
+		width: 100vw !important;
+	}
 }
 
 @media (min-width: 1024px) {
-    .application-dropdown {
-        max-width: 38rem;
-    }
+	.application-dropdown {
+		max-width: 38rem;
+	}
 
-    .dashboard-background-layout {
-        width: calc(100vw - 16rem) !important;
-    }
+	.dashboard-background-layout {
+		width: calc(100vw - 16rem) !important;
+	}
```

Comment:
Could you please check your format as well? Many changes in this file and the UserDataTable file are format change, which we shouldn't change them.